src/musicbrainz/tagfetcher.cpp

```diff
@@ -27,6 +27,9 @@
 #include <QUrl>
 #include <QtConcurrentMap>
 
+const std::array<QString, 5> TagFetcher::kFetchedFields{
```

Comment:
Have you any objections adding the methods
```
QVariant get_attribute( AttributeId ) const;
void set_attribute( AttributeId, const QVariant& );
```
to the Song class and make AttributeId a nested enum? My feeling is, that it logically belongs there and may be useful somewhere else.